static/js/jira-configuration.js

```diff
@@ -16,6 +16,20 @@ $('.add-organization-link').click(function (event) {
   }, 100)
 })
 
+$('.add-ghae-organization-link').click(function (event) {
+  event.preventDefault()
+  const githubHost = ((document.getElementById("githubHost")||{}).value)||"";
+  const child = window.open(`${appUrl}/github/login?jwt=${encodeURIComponent(params.get('jwt'))}&xdm_e=${encodeURIComponent(params.get('xdm_e'))}&githubHost=${encodeURIComponent(githubHost)}`)
```

Comment:
if jwt or xdm_e not there in request will it throw 500? handle errors

Reply:
`jwt` value will not be used Immediately in (`github-oauth.js`) login call. it will be passed along on all jira cloud operations and handled there in those calls. similar for `jiraHost`

but since I have used `jiraHost` in `github-oauth.js`) login call as well will handle error there